ide/vs-code/README.md

```diff
@@ -0,0 +1,10 @@
+# VS Code
+
+Download: <https://code.visualstudio.com/>
```

Comment:
In `.gitconfig` under `[core]`, you can add `editor = code --wait` to configure Git to open things into VS Code:

```
[core]
  editor = code --wait
```